utils/gyb_syntax_support/__init__.py

```diff
@@ -179,3 +180,27 @@ def _digest_trivia(trivia):
         _digest_trivia(trivia)
 
     return digest.hexdigest()
+
+
+def create_syntax_buildable_expressible_as_syntax_collection_conformances():
+    collection_map = {}
+
+    for node in SYNTAX_NODES:
+        if node.is_syntax_collection():
+            expressible_as_syntax_collection = 'ExpressibleAs' + \
+                node.syntax_kind
+
+            expressible_as_child = syntax_buildable_child_type(
+                node.collection_element_type, 
+                node.collection_element, 
+                node.is_token())
+
+            protocols = collection_map.get(expressible_as_child)
+
+            if protocols is None:
+                protocols = set([])
+
+            protocols.add(expressible_as_syntax_collection)
+            collection_map[expressible_as_child] = protocols
```

Comment:
I think a cleaner pattern would be

```suggestion
            if expressible_as_child not in collection_map:
              collection_map[expressible_as_child] = []
            
            collection_map[expressible_as_child].add(expressible_as_syntax_collection)
```

Also, I would rather not use `set` to get deterministic iteration order. Otherwise, just re-running gyb might generate different files.